packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -170,9 +177,30 @@ const DismissTipButton = () => {
   )
 }
 
+/**
+ * When the screen is small, the margin between the send tip and dismiss
+ * buttons container and the right-hand side of the tile becomes smaller.
+ * When this is smaller than the margin we want to allow, we use the short
+ * version of the button which does not include the name.
+ */
+const MIN_USERS_TO_BUTTONS_MARGIN = 8
+
 export const FeedTipTile = () => {
   const isTippingEnabled = getFeatureEnabled(FeatureFlags.TIPPING_ENABLED)
 
+  const { position: containerPosition, ref: containerRef } = usePosition()
+  const { position: buttonsPosition, ref: buttonsRef } = usePosition()
```

Comment:
Hello! drive by comment. we do this elsewhere using media queries from JS. I think more performant and potentially cleaner:

https://github.com/AudiusProject/audius-client/blob/5492cb20da8dee8f3aa0d70fa8c37dca062010a0/packages/stems/src/components/Button/hooks.ts#L22

(see behavior when collapsing UI on the track page)

Reply:
interesting. thanks will look